recipe/meta.yaml

```
@@ -13,15 +13,17 @@ source:
   sha256: {{ tar_gz_hash }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - cmake
     - {{ compiler("fortran") }}        # [linux or osx]
     - make                             # [linux or osx]
+    - m2w64-make                       # [win]
     - {{ compiler("m2w64_fortran") }}  # [win]
-    - m2w64-toolchain                  # [win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ stdlib("c") }}                # [win]
```

```suggestion
    - {{ stdlib("m2w64_c") }}                # [win]
```
For MinGW you should use value for the `stdlib`